src/clients/auth0_client.rs

```diff
@@ -37,6 +37,10 @@ impl FromConn for Auth0Client {
     }
 }
 
+fn generate_password() -> String {
+    Alphanumeric.sample_string(&mut rand::thread_rng(), 60)
+}
```

Comment:
[Our stylistic convention in Janus](https://github.com/divviup/janus/blob/main/docs/DEVELOPMENT.md#code-style) would be to do this as an `impl Distribution<Password> for Standard`, as in [`AuthenticationToken`](https://github.com/divviup/janus/blob/5ed3b3ca9f323d357374b03d81ca535ae97ef188/core/src/task.rs#L571). You'd also then need a type like `struct Password(String)` so you'd have something to hang the implementation on, which might not be a bad idea anyway.

Reply:
What's the advantage of this over the existing Distribution of Alphanumeric?

Reply:
It's just a stylistic convention we landed on. TBH I imagine there are several places where this project style isn't in line with `janus` policies so I think we could move forward with this as it is, and consider aligning style with other Divvi Up projects in the future.

Reply:
The advantage of `impl Distribution<Password> for Standard`, if `Password` is an existing type, is that this is how the `rand` crate "expects" to be used & it allows this type to fit in better with other uses of the API -- various other places in the 

For example, one could type `let password: Password = random()` (or even `let password = random()` if the context is clear enough for rustc & human readers to infer the type). Or one could generate a vector of passwords via `sample_iter`, etc.

The implementation would be almost exactly like the body of `generate_password` currently, except that the `Distribution<Password>` version gives you an `Rng` to use instead of using `thread_rng` (in practice, I believe this RNG will be a thread_rng in most cases).

(I'm not sure a `Password` type is justified, but I wouldn't object to it, and if it's there we should certainly `impl Distribution<Password>` rather than rolling-our-own convention.)